packages/libsolv/libsolv.spec

```diff
@@ -232,10 +235,16 @@ set -ex
   -DPYTHON_EXECUTABLE=%{__python2}               \
 %if %{with python3_bindings}
   -DENABLE_PYTHON3=ON                            \
-  -DPYTHON3_EXECUTABLE=%{__python3}              \
+  -DPYTHON_EXECUTABLE=/usr/bin/python3.11        \
+  -DPYTHON_LIBRARY=/usr/lib64/libpython3.11.so.1.0 \
+  -DPYTHON_INCLUDE_DIR=/usr/include/python3.11   \
```

Comment:
I think this shouldn't be needed if you use the PYTHON3 variant: https://github.com/openSUSE/libsolv/blob/23cbed3219bd07b5c3fa1ed8a6f2fa6c478c0fdb/bindings/python3/CMakeLists.txt#L14 does this.

Reply:
When I tried with python3, it built using python 3.9 on EL9, I'm really no sure whats going on there

Reply:
You may not have set `PYTHON3_EXECUTABLE` so it was using `/usr/bin/python3` instead of `/usr/bin/python3.11`.